jenkins-build-pack-api/Jenkinsfile

```diff
@@ -727,6 +809,47 @@ def loadBuildModules(buildModuleUrl){
 		environmentDeploymentMetadata = load "environment-deployment-metadata-loader.groovy"
 		echo "Environment deployment data loader module loaded successfully."
 
+		apigeeModule = load "apigee-module.groovy"
```

Comment:
Can we load this only when apigee integration is enabled? installer-vars.json should have a flag to turn this integration on/off.

Reply:
updated